ci/testgrid/config.yaml

```diff
@@ -125,6 +128,10 @@ dashboards:
     test_group_name: ci-knative-serving-api-coverage
     description: 'Conformance tests API coverage.'
     base_options: 'exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name='
+  - name: performance
```

Comment:
now we have both Latency and Performance, might cause confusion. Should we rename Latency to something else?

Reply:
maybe we can rename latency to E2E latency or something like that to indicate that the numbers are from e2e tests?

Reply:
how about "E2E Metric" vs. "Perf Benchmark"?